.travis.yml

```diff
@@ -51,8 +51,8 @@ install:
   - luarocks install --local lpeg
   - luarocks install --local lua-cjson
   - eval $(luarocks path)
-  - rvm install --binary 2.1.4
-  - rvm use 2.1.4
+  - rvm install --binary 2.3.1
```

Comment:
Do we need to update the `CHEFDK_GECODE_PATH` for the new version number?  (L39)